openassessment/xblock/static/js/src/lms/oa_staff_area.js

```diff
@@ -191,6 +230,34 @@
         },
 
         /**
+         * Closes the staff grade/assessment section of the staff area.
+         *
+         * @param {boolean} clear if true, remove the staff grade form and collapse it. Otherwise
```

Comment:
Tiny nit: Can we capitalize this first `if` here, or have some other way of indicating separation between the variable name and the following sentence? Reading `clear if true` tripped me up the first time I read it.